### PR TITLE
Branch 2

### PR DESCRIPTION
**Pull request by mkpi-pm**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+process.env.NODE_ENV = "test";
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const db = require("../db/connection.js");
+const app = require("../app");
+const request = require("supertest");
+const { string } = require("pg-format");
```
Comment (popatre) [inline __tests__/app.test.js]: remove this

**Review by popatre — APPROVED** [review]
Change this, then merge

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+});
+
+describe("/api", () => {
+  describe("/api/topics", () => {
+    test("200: Returns an array of topic objects, each with slug and description properties", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body: { topics } }) => {
+          // console.log(topics, "from app.test");
```
Comment (mallionaire) [inline __tests__/app.test.js:23]: Please remove old console.logs - we'll be really strict on this throughout the week.

File: app.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./db/controllers/news.controller");
+
+app.use(express.json());
```
Comment (mallionaire) [inline app.js:5]: Leave this out until you actually need to deal with POST requests please

File: db/controllers/news.controller.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+const topics = require("../data/test-data/topics");
```
Comment (mallionaire) [inline db/controllers/news.controller.js:1]: We don't need the topic data in this file 😄

File: db/models/news.model.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+const db = require("../connection");
+
+exports.fetchTopics = () => {
+  return db.query("SELECT * FROM topics").then((response) => {
+    // console.log(response.rows);
```
Comment (mallionaire) [inline db/models/news.model.js:5]: remove console.logs please

**Review by mallionaire — COMMENTED** [review]
Fab start! 
You appear to have accidentally nested your models and controllers folders inside the db folder. Please extract these as soon as possible - its likely to cause problems the longer you leave them there, and don't forget to update any file paths!

I would recommend getting into the habit of error handling as you go, eg. what would happen if I tried the endpoint `/api/tropical` - you may wish to add some standard error handling for 404: Route not found.

**Pull request merged**